arcgis-ios-sdk-samples/Route & Directions/Find closest facility to an incident (interactive)/README.metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "category": "Routing & Directions",
+    "category": "Routing and Directions",
```

Comment:
@yo1995 This category should be Route and Directions, to match the other samples in this category.

Reply:
Sorry for the miscue - a hotfix is at #917